azure-pipelines-ci.yml

```diff
@@ -4,6 +4,8 @@ pr:
   branches:
     include:
       - '*'
+    exclude:
+      - infrastructure
```

Comment:
This should be a path not a branch exclude:

```
pr:
  branches:
    include:
      - '*'
  path:
    exclude:
      - infrastructure
```
https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#pr-autocancel-branches-paths-drafts

Reply:
should that be include: infrastructure?
about to put a PR for azure-pipelines-ci.yml...

`pr:
  branches:
    include:
      - '*'
  paths:
    exclude:
      - 'infrastructure'`

Reply:
sorry - ignore. I thought this was for infra-ci